package.json

```diff
@@ -39,7 +39,6 @@
     "karma-mocha": "^0.1.10",
     "karma-sinon": "^1.0.4",
     "mocha": "^2.1.0",
-    "ng-annotate": "^0.15.4",
```

Comment:
sorry, deleted in favor of adding gulp-plugins later.  this is what Jon Pappa was using in his example.